configuration/rules-blockly-voice-and-multimedia.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: documentation
+title: Rules Blockly - Voice and Multimedia
+---
+
+# Multimedia
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+## Introduction
+
+In general, this section deals with two topics
+
+1) Playing or streaming audio to an audio sink e.g a speaker
+2)  Saying a text via using any Text-to-Speech API (e.g. Google's API)
+
+It should be noted that both functionalities do not just work right out of the box because e.g for (1) the audio sinks need to be installed in openHAB and for (2) e.g. the whole Google API needs to be configured in particular with your own personal google account. Even though (2) is not trivial but after having done that the usage of the functionality with blockly is very easy and worth the effort.
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.